packages/lib/tagOnResponse/cache.ts

```diff
@@ -0,0 +1,27 @@
+import { revalidateTag } from "next/cache";
+
+interface RevalidateProps {
+  id?: string;
+  responseId?: string;
+  functionName?: string;
+}
+
+export const tagOnResponseCache = {
+  tag: {
+    byResponseIdAndId(responseId: string, id: string) {
```

Comment:
It should be `byResponseIdAndTagId` (tagId instead of id)

Reply:
Done